### PR TITLE
MyDomain-avatar

### PR DESCRIPTION
**Pull request by YuliaMiha**

### CODE REVIEW

**Comment by YuliaMiha** [comment]
image_2023-06-22_21-06-30
image_2023-06-22_21-06-51

File: common/components/ProfilePage/index.tsx [hunk 1]
```diff
@@ -117,6 +128,19 @@ const ProfilePage: React.FC = () => {
                 placeholder="Введіть адресу"
               />
             </Card>
+            <Card size="small" title="Мій домен" className={s.Edit}>
+              {userDomains.map((domain) => {
```
Comment (myershov) [inline common/components/ProfilePage/index.tsx:132]: .filter
.map

File: common/components/ProfilePage/index.tsx [hunk 2]
```diff
@@ -15,17 +15,28 @@ import UnsavedChangesModal from '../UI/UnsavedChangesModal'
...
-  const user = userData
+  let user: any
+  if (userData) {
+    user = userData
+  }
+
+  const { data: domainData, isLoading: isDomainLoading } = useGetDomainsQuery({
+    streetId: '',
+    limit: undefined,
+    adminEmail: user?.email ? user?.email[0] : undefined, // Pass the first email from the adminEmails array to fetch domains associated with the user
```
Comment (myershov) [inline common/components/ProfilePage/index.tsx:36]: remove

File: common/api/domainApi/domain.api.ts [hunk 3]
```diff
@@ -19,12 +19,16 @@ export const domainApi = createApi({
...
     >({
-      query: ({ limit, streetId }) => {
+      query: ({ limit, streetId, adminEmail }) => {
         return {
           url: `domain`,
-          params: { limit, streetId },
+          params: {
+            limit,
+            streetId,
+            adminEmail,
```
Comment (myershov) [inline common/api/domainApi/domain.api.ts:30]: remove

**Comment by YuliaMiha** [comment]
image
image

**Pull request merged**